packages/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.0
+
+* Breaking change: Path of returned File objects when picking videos does no longer start with `file:///`, but are instead a normal absolute path.
```

Comment:
please specify it is on iOS platform. Also the path starts with `file:///` is the absolute path, you might want to update the wording a little bit?

Reply:
Is this better?

> Breaking change iOS: Returned `File` objects when picking videos now always holds an absolute path. Before this change, a `File` object with a path beginning with `/file:///` could be returned.

Reply:
Maybe 
```Breaking change iOS: Returned `File` objects when picking videos now always holds the correct path. Before this change, the absolute path (Begin with file:///) was returned.```

Reply:
> (Begin with file:///)

just to clarify, the path returned on the `File` object (e.g. `print(file.path)`) starts with a single forward slash, and then `file:`, followed by three forward slashes. At least on my iPhone 8, iOS 12.2.

This is the code that we are currently using in our app to work around it:

```dart
      if (file.path.startsWith('/file:///')) {
        file = File(file.path.substring(8));
      }
```

Reply:
Changed to

> Breaking change iOS: Returned `File` objects when picking videos now always holds the correct path. Before this change, the path returned could have `file://` prepended to it.

What do you think?

Reply:
> > (Begin with file:///)
> 
> just to clarify, the path returned on the `File` object (e.g. `print(file.path)`) starts with a single forward slash, and then `file:`, followed by three forward slashes. At least on my iPhone 8, iOS 12.2.
> 
> This is the code that we are currently using in our app to work around it:
> 
> ```dart
>       if (file.path.startsWith('/file:///')) {
>         file = File(file.path.substring(8));
>       }
> ```

yeah that was a typo of mine, you can change it to /file:/// in the changelog.